### ISSUE TITLE
Support for Ecobee4

### ISSUE THREAD

**Issue by fratinize** [vojtamolda/homebridge-ecobee3-sensors#26]
Is Anybody Supporting this? Can We Add Ecobee4? Thanx

**Comment by vojtamolda** [comment]
Hello @fratinize !

Nope. Nobody actively supports it. The repos is my hobby open source project. I don't plan to work on adding support for Ecobee 4.

I'll be happy to merge your PR, though. From what I know you shouldn't need to change many things. Ecobee 4 only added support for voice commands and the API is identical.

**Comment by fratinize** [comment]
Ty for the quick response
I did read through the enhancement issue from virtualboy
Found the line 231 replaced the athenaSmart with the apolloSmart
But now hb won’t start.
Forgive my noobness first time digging this deep 
in my logs I see several lines mentioned? Am I to understand I need to check those lines and correct them?
I am not really sure how to go about a PR and how it’s different then changing existing files?🤔☹️
Thanx for any direction

**Comment by vojtamolda** [comment]
No worries about noobness :) Everyone was a noob before. Without realizing it I actually meant that the changes would be easy for a developer. I'm not sure what software development experience you have and what's your background. There may be some learning involved in order to get the support for Ecobee 4 implemented.

I'm a quite curious about why do you need this plugin. It was developed in early days after the release of Ecobee 3. It initially didn't expose the wireless temperature/occupancy sensors to HomeKit so this plugin fixed that. About a year later they released a firmware update and now the individual sensors are accessible in the Home app by default for everyone.

Anyway, if you still want to hack around, I'd start with some general node.js tutorials. It's a simple way to learn how to debug an app and see what it is doing. YouTube works pretty well and you can findy plenty of tutorial videos. PRs are a way to propose changes to repositories on GitHub and again I'd start by watching a video and trying it out.

**Comment by fratinize** [comment]
Honestly Not Sure why I went down this road. I had Just added 2 New Sensors to my Ecobee and Did not see them Right away in Home Kit. Didn’t Really Think the Whole Thing out When I raised the Flag. Basically failed to do proper research :(

I was abe to Add the 4 using the addition of the apolloSmart to the Platform.js That was Educational. :)

Sorry about that 

Thank You for the Tips Though.


Regards
Frank
